test/e2e.integration.test.js

```diff
@@ -236,6 +236,28 @@ test('ledger : user contribution workflow with uphold BAT wallet', async t => {
   t.true(balance.length > 1)
   t.is(+balance, 0)
 
+  const select = `
+SELECT *
+FROM transactions
+WHERE
+    transaction_type = 'contribution_settlement';
+`
+  const {
+    rows
+  } = await postgres.query(select)
+
+  t.deepEqual(rows.map((entry) => _.omit(entry, ['from_account', 'to_account', 'document_id', 'id'])), [{
+    created_at: new Date(+newYear + 2000),
```

Comment:
looks like `newYear` isn't defined anywhere?

Reply:
yeah. my bad. not sure how that got through the commit. just uploaded